docs/paper/evidence/web-2026-04-06/luminair_post.html

```
@@ -0,0 +1,126 @@
+                // scriptElement.setAttribute("async", "");
+                // scriptElement.setAttribute("defer", "");
+                scriptElement.setAttribute("src", "//js-eu1.hs-scripts.com/26199107.js");
+                scriptElement.appendChild(document.createTextNode(``));
+                document.head.appendChild(scriptElement);
+            }, 5000);
+        </script>
+
+        <!-- End of HubSpot Embed Code -->
+
+<div id="gsp_data_html" data-g_version="3.19.5" data-w_id="08e83d8739e6f4bdef27dbfedcf41b67" data-s_enc="" data-h_enc="" data-hh_enc="" data-lazy_load="1" data-reload_session="0" data-gt-w="gsp_clgtranslate_wrapper" data-vv_index="25" data-ref=""></div>
+</body>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </body> ]

(tag-pair)